servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -708,6 +708,12 @@ public Builder successfulActiveHealthCheckUnejectHost(final boolean successfulAc
     }
 
     static boolean enforcing(int enforcingPercentage) {
-        return enforcingPercentage >= 100 || ThreadLocalRandom.current().nextInt(100) <= enforcingPercentage;
+        if (enforcingPercentage == 0) {
```

Comment:
should we enforce a valid range regardless?

Reply:
I don't know that we want to blow up with a wrong value. I have made it more clear that values less than 0 are interpreted as 0.